### ISSUE TITLE
LAST parameter selection

### ISSUE THREAD

**Issue by luoxixi666** [mcfrith/last-genome-alignments#22]
Hello, I have a problem with the last parameter selection now, my purpose is to use the obtained maf file to draw circos collinearity alignment diagram, but the drawn graph has not been very good, and the obtained maf file is also very large, the following is my process: lastdb -c -P 20 -u NEAR mydb ./re.genoma.fas, 
lastal -P 20 -E 0.05 ./mydb genoma.fas./ > output.maf,
last-split output.maf > final.maf, 
my genome is mammal, so need  -E 0.05 can you give me some advice? Thank you

**Comment by mcfrith** [comment]
Yes, the parameters are too complicated, sorry. For aligning 2 genomes, the latest parameter recommendations are here:
https://gitlab.com/mcfrith/last/-/blob/main/doc/last-cookbook.rst

See "Aligning human & chimp genomes" and "Aligning more distantly related genomes".

Recently, I recommend `-D` (or maybe `-H`) instead of `-E`. They are basically different ways of setting the same thing (score / e-value threshold), but I now think `-E` is less intuitive.

Hope that helps!

**Comment by luoxixi666** [comment]
> Yes, the parameters are too complicated, sorry. For aligning 2 genomes, the latest parameter recommendations are here: https://gitlab.com/mcfrith/last/-/blob/main/doc/last-cookbook.rst
> 
> See "Aligning human & chimp genomes" and "Aligning more distantly related genomes".
> 
> Recently, I recommend `-D` (or maybe `-H`) instead of `-E`. They are basically different ways of setting the same thing (score / e-value threshold), but I now think `-E` is less intuitive.
> 
> Hope that helps!

Thank you so much for your reply, what is the specific step you run, and what is the specific value of -D? We look forward to hearing from you again！

**Comment by mcfrith** [comment]
You can try the specific steps at that link (which use `-D1e9`).

**Comment by luoxixi666** [comment]
> You can try the specific steps at that link (which use `-D1e9`).

Excuse me, are you using LAST version 1595, the latest version?

**Comment by mcfrith** [comment]
Yes!

**Comment by luoxixi666** [comment]
> Yes!
Thank you for your reply, this is the circos diagram that I just drew with the running maf after replacing E with D, it still looks very messy, but the kinship of these two mammals is very close, it stands to reason that it shouldn't be so messy, do you have any other good suggestions, really thank you so much
![ DE9 chr](https://github.com/user-attachments/assets/932f8c70-0793-42a9-aef3-57ade5906cda)

**Comment by mcfrith** [comment]
I'm not sure if you **only** replaced E with D, in the recipe from your initial message.

I suggest the steps here: https://gitlab.com/mcfrith/last/-/blob/main/doc/last-cookbook.rst

I'm not sure if you did this already:
* Get many-to-one alignments
→ `last-split -r`
→ one-to-one alignments
→ `maf-linked`
